pic-search.html

```diff
@@ -1,157 +1,131 @@
-                <a
-                  href=""
-                  class="uk-slidenav uk-slidenav-next"
-                  data-uk-slideset-item="next"
-                ></a>
-              </div>
+<body>
+  <!-- Navigation Bar Visible on Large Viewports-->
+  <nav class="uk-navbar">
+    <div class="uk-container uk-container-center">
+      <ul class="uk-navbar-nav">
+        <li class="uk-parent uk-visible-large">
+          <a href="index.html">Home</a>
+        </li>
+        <li class="uk-parent uk-visible-large">
+          <a href="developers.html">Developers</a>
+        </li>
+      </ul>
+      <!-- Hamburger Menu Button Hidden on Large Viewports-->
+      <button id="sidebar-button" href="#sidebar" class="uk-button uk-navbar-toggle uk-hidden-large"
+        data-uk-offcanvas="{target:'#sidebar'}"></button>
+    </div>
+  </nav>
+  <div></div>
+  <!-- Slideset -->
+  <div class="slideset-wrapper">
+    <div class="uk-container uk-container-center">
+      <div class="uk-grid">
+        <div class="uk-width-1-1">
+          <!-- Slideset -->
+          <div data-uk-slideset="{small: 1, medium: 2, large: 3, animation: 'slide-horizontal', duration: 200}">
+            <div class="uk-slidenav-position">
+              <ul class="uk-grid uk-slideset">
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+              </ul>
+              <a href="" class="uk-slidenav uk-slidenav-previous" data-uk-slideset-item="previous"></a>
+              <a href="" class="uk-slidenav uk-slidenav-next" data-uk-slideset-item="next"></a>
             </div>
           </div>
-          <!-- column -->
         </div>
-        <!-- grid -->
+        <!-- column -->
       </div>
-      <!-- container -->
+      <!-- grid -->
     </div>
-    <!-- slideset wrapper -->
-
-    <!-- Mobile Sidebar -->
-    <div id="sidebar" class="uk-offcanvas">
-      <div class="uk-offcanvas-bar uk-offcanvas-bar-flip">
-        <ul class="uk-nav uk-nav-offcanvas">
-          <li class="uk-active"><a href="">Home</a></li>
-          <li class="uk-parent"><a href="">Developers</a></li>
-        </ul>
-      </div>
+    <!-- container -->
+  </div>
+  <!-- slideset wrapper -->
+  <!-- <div class="uk-container uk-container-center">
+    <img src="" alt="quotePic-final" id="final-product">
+    <!-- <canvas id="canvas" style="border: .5px solid black;" height="830px" width="553">Sorry. Canvas not supported. -->
+  <!-- Try newest version of Google Chrome.</canvas> -->
+  </div> -->
+  <!-- Mobile Sidebar -->
+  <div id="sidebar" class="uk-offcanvas">
+    <div class="uk-offcanvas-bar uk-offcanvas-bar-flip">
+      <ul class="uk-nav uk-nav-offcanvas">
+        <li class="uk-active"><a href="./index.html">Home</a></li>
```

Comment:
I added these links and also changed the name of the buttons; fix how you want to